packages/terra-date-picker/CHANGELOG.md

```diff
@@ -15,6 +15,7 @@ Unreleased
 * Added clinical-lowlight-theme theming files
 
 ### Changed
+* Fixed issue due to two dates getting selected simultaneously.
```

Comment:
Can we elaborate the other changes listed [here](https://github.com/cerner/terra-framework/pull/994#issuecomment-587482809)? Multiple log entries can be added if doing so makes it easier to call out these sets of changes.

Reply:
Elaborated here f91e565